### ISSUE TITLE
Inconsistent Responses

### ISSUE THREAD

**Issue by siunus** [EvilFreelancer/routeros-api-php#121]
**Version of RouterOS**
RouterOS 6.48.3 (stable)

**To Reproduce**
Running a query like the one below sometimes yields unexpected responses:
```php
$query = (new Query('/ppp/secret/print'));
$res = $client->query($query)->read(false);
```

**Expected behavior**
The correct response should look like this:
![image](https://github.com/user-attachments/assets/f15c1dc1-1df5-47c8-b1dd-b0f76636fe35)

However, after refreshing a couple of times, I got this instead:
![image](https://github.com/user-attachments/assets/e9573cef-92d9-4824-99cb-5612d884b3a2)

The parsed result is cut off in this section:
![image](https://github.com/user-attachments/assets/38b52068-88bf-4d50-b08d-a63110ff0fed)

Could someone explain where the problem might be?

**Comment by EvilFreelancer** [comment]
Hi! Looks like a bug on the RouterOS side, which version of ROS do you use? Did you tried other versions?

**Comment by somzoli** [comment]
I have the same issue here. /export results sometimes missing the end of the full export. If i run the export on router terminal, the output is full, but when i run the qury in router-os api, the results sometimes not contain the last X line. 
We have huge configurations, so the export contains about ~60000 line.

**Comment by somzoli** [comment]
In ssh (from my pc), the export works as expected. Seems its a [routeros-api-php] issue.

**Comment by somzoli** [comment]
For export (ssh), ive made a pull request for change executeAsync to execute. In ssh export this caused early quit from loop.

**Comment by EvilFreelancer** [comment]
Hello! Thanks, PR merged, release tagged as 1.5.4
